js/ui/components/clubhouse.js

```diff
@@ -678,13 +590,17 @@ class ClubhouseNotificationSource extends NotificationDaemon.GtkNotificationDaem
 
 var Component = GObject.registerClass({
 }, class ClubhouseComponent extends SideComponent.SideComponent {
-    _init() {
-        super._init(ClubhouseIface, CLUBHOUSE_ID, CLUBHOUSE_DBUS_OBJ_PATH);
+    _init(clubhouseIface, clubhouseId, clubhousePath) {
```

Comment:
Would it be easier to have just an argument isOldClubhouse? If so, I would prefer that.